services/membership/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 > This module has been replaced with the authorization module, which connects to the same API.
 >
 > **This module will receive no further updates.** Use `github.com/stackitcloud/stackit-sdk-go/services/authorization` instead.
+## v0.5.0 (2024-04-09)
```

Comment:
this module should not be updated, it is deprecated.

The SDK generation should be skipped, but for now just remove it from the PR